site/src/pages/AIBridgePage/RequestLogsPage/RequestLogsRow/RequestLogsRow.tsx

```diff
@@ -21,6 +25,47 @@ type RequestLogsRowProps = {
 	interception: AIBridgeInterception;
 };
 
+const customisedDateLocale: Intl.DateTimeFormatOptions = {
+	// Hide the year from the date
+	year: undefined,
+	// Show the month as a short name
+	month: "short",
+	day: "numeric",
+	hour: "2-digit",
+	minute: "2-digit",
+	second: "2-digit",
+	hour12: true,
+};
+
+const RequestLogsRowProviderIcon = ({
```

Comment:
this is the second place where we're switching on the provider name for an icon, we should definitely consolidate that